cli/crates/graphql-introspection/src/standard.rs

```diff
@@ -0,0 +1,30 @@
+use parser_graphql::ConnectorHeaders;
+use url::Url;
+
+use crate::Error;
+
+pub(super) async fn introspect(url: &Url, headers: &[(&str, &str)]) -> Result<String, Error> {
+    let result = parser_graphql::parse_schema(
+        reqwest::Client::new(),
+        "introspection",
+        false,
+        url,
+        ConnectorHeaders::default(),
+        headers.iter().map(|(key, value)| (*key, *value)),
+        None,
+    )
+    .await;
+
+    match result {
+        Ok(registry) => Ok(registry.export_sdl(true)),
```

Comment:
Is there a reason to use `parser_graphql` & round-trip via a `Registry` with this?

I wouldn't necessarily trust it to not add GraphQL connector specific bits to the schema.

Reply:
Good point. Should we use some standard mechanism, such as cynic here?

Reply:
Yeah, might be a safer bet to just use it directly instead of going via `parser_graphql`.  [It's relatively easy to do introspection->SDL](https://github.com/obmarg/cynic/blob/main/cynic-cli/src/introspect.rs#L18-L32)